nikola/plugin_manager.py

```diff
@@ -35,13 +35,12 @@
 from collections import deque
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Dict, List, Optional, Type, TYPE_CHECKING, Set
+from typing import Dict, Iterable, List, Optional, Type, Set
 
 from .plugin_categories import BasePlugin, CATEGORIES
 from .utils import get_logger
 
-if TYPE_CHECKING:
-    import logging
+import logging
```

Comment:
Consider moving this to the top (line 32).

Reply:
Done.

(I'm used to `isort` doing this kind of thing for me.)